build.gradle

```diff
@@ -95,6 +96,11 @@ dependencies {
     implementation 'jstl:jstl:1.2'
     // http://commons.apache.org/proper/commons-configuration/dependencies.html
     implementation 'commons-beanutils:commons-beanutils:1.9.3'
+    implementation 'com.zaxxer:HikariCP:3.3.1'
+    // https://mvnrepository.com/artifact/org.flywaydb/flyway-core
+    implementation group: 'org.flywaydb', name: 'flyway-core', version: '7.2.0'
```

Comment:
These two dependencies could (and should) be `testImpementation`.

Reply:
Correct, I'm using them only for tests.